apps/testing/20-modal/src/app/app.component.cy.ts

```diff
@@ -1,19 +1,41 @@
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { AppComponent } from './app.component';
 
 describe(AppComponent.name, () => {
   const setup = () => {
-    cy.mount(AppComponent);
+    cy.mount(AppComponent, { imports: [BrowserAnimationsModule] });
   };
 
-  test('error modal is displayed if you click on "Confirm" without inputing a name', () => {
+  test('error modal is displayed if you click on "Confirm" without inputting a name', () => {
     setup();
+
+    cy.contains('button', 'Confirm').click();
+
+    cy.get('mat-dialog-container div').should(
```

Comment:
I don't think you have to specify the extra div. should.contain will look inside the DOM node.